config/c-code/manylinux-install.sh.j2

```diff
@@ -30,13 +30,14 @@ yum -y install libffi-devel
 tox_env_map() {
     case $1 in
 {% if with_future_python %}
-        *"cp313"*) echo 'py313';;
+        *"cp314"*) echo 'py314';;
 {% endif %}
         *"cp38"*) echo 'py38';;
         *"cp39"*) echo 'py39';;
         *"cp310"*) echo 'py310';;
         *"cp311"*) echo 'py311';;
         *"cp312"*) echo 'py312';;
+        *"cp313"*) echo 'py313';;
```

Comment:
Maybe it is a good point in time to compute these lines (and the other ones referring to Python versions) from the values set in `shared/packages.py`. This would make it easier to add/remove Python versions. What do you think?

Reply:
IMHO it would be great to compute the selection of Python versions in all places where we want to say "all Python versions between the declared minimum and maximum", such as in `tox.ini` as well. I'd actually like to be in a place where I can more easily use zope.meta outside of zopefoundation packages, for example with work-related packages where I want to say "my minimum is Python 3.11".

Reply:
I have converted all hardcoded Python versions lists so they are generated by a loop.